src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.stories.tsx

```diff
@@ -51,15 +51,9 @@ const sampleEndorsementChain: (TradeTrustErc721Event | TitleEscrowEvent)[] = [
   },
   {
     documentOwner: "0xBee0875Ba8069ed5c48E6A670118EF1C6B1E7fC0",
-    eventType: "Transfer",
+    eventType: "Transfer to Wallet",
```

Comment:
think this 1 still got the spare dash in holder

Reply:
ok